acquia/acquia_cms
# ACMS-1730: Provide acquia recommended configuration management commands.

**Motivation**

Fixes ACMS-1730
**Proposed changes**
- Creating a acquia global command for configuration management.
- Add event subscriber to post config command in headless and site studio.

**Testing Steps:**
Verify Headless and Site Studio config Export and Import.
- Update `./modules/acquia_config_management/composer.json` line 8 from `"acquia/acquia_global_commands": "dev-main"` to `"acquia/acquia_global_commands": "dev-ACMS-1730"`
- Require global command drush plugin using `composer require acquia/acquia_global_commands:dev-ACMS-1730`.
- Use ACMS-1730 branch to install site using `./vendor/bin/drush si minimal --account-pass=admin`
- Install Acquia Config Management module `./vendor/bin/drush in acquia_config_management -y`
- **for headless testing**: Install ACMS headless module `./vendor/bin/drush in acquia_cms_headless -y`
- **for site studio testing**: export SS keys and Install ACMS Site Studio module `./vendor/bin/drush in acquia_cms_site_studio_cm -y`
- Export config `./vendor/bin/drush cex -y`
- **for site studio testing**: Verify site studio packages exported at `../config/<site_name>/sitestudio`
- Copy uuid using `./vendor/bin/drush cget system.site uuid` then copy the ID from output
- Install site `./vendor/bin/drush si minimal --account-pass=admin`
- Set site uuid `./vendor/bin/drush cset system.site uuid <copied uuid>`
- Import config `./vendor/bin/drush cim -y && ./vendor/bin/drush cr && ./vendor/bin/drush cim -y`
- **for headless testing**: Verify Headless user created after cim
- **for site studio testing**: Verify site studio packages imported after cim
